src/__snapshots__/Skedify.test.js.snap

```diff
@@ -725,3 +767,39 @@ exports[`API/Config should throw an error when onConfigurationChange is not pass
 exports[`API/Config should throw an error when overriding items on the configuration object directly 1`] = `"Cannot assign to read only property 'locale' of object '#<Object>'"`;
 
 exports[`API/Config should throw an error when overriding the whole config object 1`] = `"[CONFIG]: You are not allowed to change the configuration object directly. Please use \`.configure({})\` instead."`;
+
+exports[`auth providers should be possilbe to use the \`public_client\` strategy 1`] = `
+Object {
+  "data": "my subjects data",
+  "errors": undefined,
+  "headers": undefined,
+  "meta": undefined,
+  "status": 200,
+  "warnings": undefined,
+}
+`;
+
+exports[`auth providers should be possilbe to use the \`token\` strategy 1`] = `
+Object {
+  "data": "my subjects data",
+  "errors": undefined,
+  "headers": undefined,
+  "meta": undefined,
+  "status": 200,
+  "warnings": undefined,
+}
+`;
+
+exports[`auth providers should error when a parameter is given that is not required or optional or the realm for a certain auth provider strategy 1`] = `
+"[IDENTITY PROVIDER]: unnecessary_key is not an optional or a required parameter for \`public_client\`
+  - Required parameters: client_id
+  - Optional parameters: scope"
```

Comment:
The realm is required for us, but the required parameters and optional parameters in here are the required / optional ones for the api. The api doesn't expect a realm. So that's why I separated them.